extensions/http/jetty-micrometer/src/main/java/org/eclipse/dataspaceconnector/extension/jetty/JettyMicrometerConfiguration.java

```diff
@@ -33,6 +34,18 @@ public JettyMicrometerConfiguration(MeterRegistry registry) {
 
     @Override
     public void accept(ServerConnector connector) {
-        connector.addBean(new JettyConnectionMetrics(registry, connector, Tags.empty()));
+        var tags = Tags.empty();
+
+        if (connector.getPort() != 0) {
```

Comment:
This condition is always true, as every `Connector` is created from a `PortMapping`, that is always created with a valid port number (checkout `JettyConfiguration.createFromConfig`)

Reply:
Reworked